radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/store/ClientApiStore.java

```diff
@@ -33,6 +35,7 @@
 /**
  * High level JSON RPC client API store.
  */
+@ImplementedBy(BerkeleyClientApiStore.class)
```

Comment:
This should be managed by Modules and not the class itself.

Reply:
Fixed